src/Domain.LinnApps/SupplierSpend.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Linn.Purchasing.Domain.LinnApps
+{
+    using System.ComponentModel.DataAnnotations.Schema;
+
+    using Linn.Purchasing.Domain.LinnApps.Suppliers;
+
+    public class SupplierSpend
+    {
+        public int PlTref { get; }
+
+        public decimal BaseTotal { get; set; }
+
+        public int LedgerPeriod { get; set; }
+
+        public int SupplierId { get; set; }
+
+        public Supplier Supplier { get; set; }
+        [NotMapped]
```

Comment:
Do these totals need to be fields on the Entity?

Reply:
I swithered on that too - they don't, but I'm keeping and using the other properties on the entity for the report so for some reason I felt like I wanted to keep using the same object and just fill in the totals fields. If you or others would prefer, it's no biggie to make another domain object for the totalled-up version. I did at least learn/remember about the [NotMapped] from it though 😄

Reply:
I'm generally in favour of trying to keep any database concerns out of the domain unless impossible, for admittedly purist/idealist reasons. 

Where do these totals actually come from? Could you not just stick them into the report results model without having them as a class field at all?